src/Makevars.win

```diff
@@ -14,7 +14,15 @@ OBJECTS = $(SOURCES:.cpp=.o) $(OBJECTS_C)
 PROJ_LIBS=$(shell "${R_HOME}/bin${R_ARCH_BIN}/Rscript" "../tools/get_proj_libraries.R")
 HDF5_LIBS=$(shell "${R_HOME}/bin${R_ARCH_BIN}/Rscript" "../tools/get_hdf5_libraries.R")
 H5_API_CFLAGS=$(shell "${R_HOME}/bin${R_ARCH_BIN}/Rscript" "../tools/get_h5_api_cflags.R")
-PKG_LIBS=-lpthread $(HDF5_LIBS) -lbz2  -lz  -ldl -lm $(PROJ_LIBS)
+
+# Conditional linking of libsharpyuv
+ifeq (,$(shell pkg-config --version 2>/dev/null))
```

Comment:
how does this relate to https://github.com/adokter/vol2birdR/pull/60, are both compatible?

Reply:
Yes, this is the version that passed all CRAN checks.